setup.cfg

```diff
@@ -55,7 +55,7 @@ install_requires =
     pip >= 20
     setuptools >= 45
     wheel >= 0.34
-    cookiecutter >= 1.0
+    cookiecutter >= 1.0,<1.7.0
```

Comment:
I'm sure there's a good reason for this - can you elaborate on what it is? Upper version pins always make me nervous, as they effectively mean the dependency is moving in a different direction to the project using it.

Reply:
This upper boundary was due to a bug in *cookiecutter*, that apparently has been fixed by now 😄 .

Removed it.